src/core/__tests__/ColorVariable_test.ts

```diff
@@ -33,7 +33,11 @@ describe("ColorVariable", () => {
   });
 
   it("have the correct datatype", () => {
-    expect(variable.dataType).to.equal(VariableType.COLOR);
+    expect(variable.dataType).to.equal(DataType.COLOR);
+  });
+
+  it("have the correct contraintType", () => {
+    expect(variable.constraintType).to.equal(ConstraintType.LIST);
```

Comment:
Yeah will do. I'll make a separate PR for that if you're ok with that. Its got a few additional tests as well as the one you're describing.